aggsender/db/aggsender_db_storage.go

```diff
@@ -564,18 +577,33 @@ func (a *AggSenderSQLStorage) GetNonAcceptedCertificate() (*NonAcceptedCertifica
 	if err := json.Unmarshal([]byte(val), &nonAcceptedCert); err != nil {
 		return nil, fmt.Errorf("failed to unmarshal non-accepted certificate: %w", err)
 	}
+	if nonAcceptedCert.SignedCertificate != "" && nonAcceptedCert.SignedCertificate[0] == '@' {
```

Comment:
Wdyt? Also it would probably make sense to extract the `@` to a const.
```suggestion
	if strings.HasPrefix(nonAcceptedCert.SignedCertificate, "@") {
```

Reply:
I think that is overcomplicating adding a const for that, but anyway I applied your suggestion